src/styles/blocks/header.scss

```diff
@@ -0,0 +1,61 @@
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.page-content__header {
+  margin-bottom: 50px;
+}
+
+.logo {
+  width: #{$logoSize};
+  height: #{$logoSize};
+}
+
+.nav {
+  &__list {
+    display: flex;
+    padding: 0;
+    margin: 0;
+  }
```

Comment:
Move this and all nav... styles to separate file.

Remembrer - each block should have its own file.